mod/wikirate_source/set/type/source/source.rb

```diff
@@ -37,10 +37,17 @@ def indirect_contributor_search_args
   end
 end
 
+def is_attached_file_exist? file_card
```

Comment:
Use def with parentheses when there are parameters.<br>Rename `is_attached_file_exist?` to `attached_file_exist?`.